### PR TITLE
PLAT-8987 - Add search date and locations filters settings

### PR DESCRIPTION
**Pull request by omaya-adm**

### CODE REVIEW

File: src/Api/Search.php [hunk 1]
```diff
@@ -100,14 +102,23 @@ public static function searchForm($args)
             if (is_admin()) {
                 return '';
             }
+            $attsArray = array(
+                'categories_filter_type' => 'select',
+                'pager' => 'simple',
+                'template' => 'grid'
+            );
+
+            $dateSettingsOption = get_option('admwpp_search_settings');
```
Comment (jadkhateradmin) [inline src/Api/Search.php]: use the method provided in the Settings class to get the option you need such as "locations_filters"

File: assets/js/admwpp.js [hunk 2]
```diff
@@ -9,5 +9,32 @@
          $('.admwpp-custom-select').selectric();
       }
 
+      var dateFormat = "mm/dd/yy",
```
Comment (jadkhateradmin) [inline assets/js/admwpp.js]: Use the global date format used in globals.php passing it to the JS with localize_scripts.
If there is none in the globals.php define one such: `define('ADMWPP_SEARCH_DATE_FORMAT', 'mm/dd/yy');`

File: templates/search/locations-filter.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
+<div class="adwmpp-filter-wrapper adwmpp-locations-wrapper locations-wrapper adwmpp-dropdown dropdown">
+    <div class="adwmpp-input-wrapper">
+        <label><i class="adwmpp-locations-icon locations-icon"></i><?php _e('locations', ADMWPP_TEXT_DOMAIN); ?></label>
+        <span class="adwmpp-chosen-options chosen-options"></span>
+        <i class="adwmpp-toggle-arrow toggle-arrow"></i>
+    </div>
+    <div class="adwmpp-dropdown-wrapper dropdown-wrapper">
+        <div class="adwmpp-dropdown-body dropdown-body">
+            <?php if ($locations_filter_type == 'select') : ?>
+                <select class="admwpp-select admwpp-custom-select" name="location">
```
Comment (jadkhateradmin) [inline templates/search/locations-filter.php]: Change the name to '**loc**' as using '**location**' will break the search page once you filter. _(did this happen on your end?)
Or maybe define it to the "queryVars" method and run some tests

**Review by jadkhateradmin — CHANGES_REQUESTED** [review]

File: templates/search/locations-filter.php (hunk 3 above)
Comment (omaya-adm) [reply templates/search/locations-filter.php]: Done

File: assets/js/admwpp.js (hunk 2 above)
Comment (omaya-adm) [reply assets/js/admwpp.js]: Done

File: src/Api/Search.php (hunk 1 above)
Comment (omaya-adm) [reply src/Api/Search.php]: Done

**Review by jadkhateradmin — APPROVED** [review]

**Pull request merged**